src/views/wallet/Validator.vue

```diff
@@ -100,24 +102,16 @@ export default class Validator extends Vue {
     }
 
     get platformLockedStakeable(): BN {
-        // return this.$store.getters.walletPlatformBalanceLockedStakeable
+        if (this.depositAndBond) return this.$store.getters['Assets/walletPlatformBalanceDeposited']
         return this.$store.getters['Assets/walletPlatformBalanceLockedStakeable']
     }
```

Comment:
Just a clarification question, this will just pass the available amount to caminojs? And caminojs will produce the correct UTXOs with choosing the deposited ones first?